tests/telepay-client.test.ts

```diff
@@ -0,0 +1,179 @@
+import { TelepayClient } from '../src';
+import {
+    CreateInvoiceBody,
+    GetWithdrawMinimumBody,
+    Invoice,
+    TransferBody,
+    WithdrawBody
+} from '../src/utils/interfaces';
+import { AxiosResponse } from 'axios';
+import { UrlHelper } from '../src/utils/enums';
+
+const TEST_SECRET_KEY = 'secret_RW182YA8LFH5T5IHIK15LTD6G9KM2CVMOU0H1YM6G21XZQ7HXEDPIBBBBIQ1Z4EQ6ME73F12NA3IGM9PDS0TJOU5E5AG8825W5U3';
```

Comment:
This must NOT be hardcoded. Let's remove this here, roll the keys and add that as repository environment secrets. Contact me and I add them.